pkg/reconciler/serverlessservice/controller.go

```diff
@@ -19,21 +19,21 @@ package serverlessservice
 import (
 	"context"
 
+	"k8s.io/client-go/tools/cache"
```

Comment:
Maybe back this change out? Seems unrelated.

Reply:
just import formatting...
I can but 🤷‍♂